pyscf/lib/numpy_helper.py

```diff
@@ -1041,6 +1045,8 @@ def condense(opname, a, loc_x, loc_y=None):
     nloc_y = loc_y.size - 1
     if a.flags.f_contiguous:
         out = numpy.zeros((nloc_x, nloc_y), order='F')
+        loc_x, loc_y = loc_y, loc_x
+        nloc_x, nloc_y = nloc_x, nloc_y
```

Comment:
is this correct?

Reply:
It's correct but seems misleading. I changed implementations in new commits